File: baekjoon/9498/9498.cpp
/*
시험 성적 C++ 풀이
https://doctcoder.tistory.com/entry/%EC%8B%9C%ED%97%98-%EC%84%B1%EC%A0%81-%ED%92%80%EC%9D%B4
*/
#include <iostream>

using namespace std;
int main() {
	int score;
	cin>>score;
	if(score>89)
	{
		cout<<"A";
	}
	else if(score>79)
	{
		cout<<"B";
	}
	else if(score>69)
	{
		cout<<"C";
	}
	else if(score>59)
	{
		cout<<"D";
	}
	else{
		cout<<"F";
	}
}

File: baekjoon/9498/9498.java
/*
시험 성적 Java 풀이
https://doctcoder.tistory.com/entry/%EC%8B%9C%ED%97%98-%EC%84%B1%EC%A0%81-%ED%92%80%EC%9D%B4
*/
import java.util.Scanner;

class Main 
{
  public static void main(String[] args)
  {
      Scanner sc=new Scanner(System.in);
      int a=sc.nextInt();
      if(a>89) System.out.println("A");
      else if(a>79) System.out.println("B");
      else if(a>69) System.out.println("C");
      else if(a>59) System.out.println("D");
      else System.out.println("F");
  }
}
